cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -1119,11 +1264,27 @@ def test_content_group_gets_indexed(self):
         )
 
         self.publish_item(self.store, self.html_unit1.location)
+        self.publish_item(self.store, self.split_test_unit.location)
 
         with patch(settings.SEARCH_ENGINE + '.index') as mock_index:
             self.reindex_course(self.store)
             self.assertTrue(mock_index.called)
             self.assertIn(self._html_group_result(self.html_unit1, [1]), mock_index.mock_calls)
+            self.assertIn(self._html_experiment_group_result(self.html_unit4, [unicode(2)]), mock_index.mock_calls)
+            self.assertIn(self._html_experiment_group_result(self.html_unit5, [unicode(3)]), mock_index.mock_calls)
+            self.assertIn(self._html_experiment_group_result(self.html_unit6, [unicode(4)]), mock_index.mock_calls)
+            self.assertIn(
+                self._vertical_experiment_group_result(self.condition_0_vertical, [unicode(2)]),
+                mock_index.mock_calls
+            )
+            self.assertIn(
+                self._vertical_experiment_group_result(self.condition_1_vertical, [unicode(3)]),
+                mock_index.mock_calls
+            )
+            self.assertIn(
```

Comment:
This needs to have some 'not in' logic too, to make sure that there are no false results. As it stands, I can't read this test and be sure that the partitioning is working as desired.